geode-connectors/src/main/java/org/apache/geode/connectors/util/internal/DescribeMappingResult.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.apache.geode.connectors.util.internal;
+
+import java.io.DataInput;
+import java.io.DataOutput;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.apache.geode.DataSerializable;
+import org.apache.geode.DataSerializer;
+
+public class DescribeMappingResult implements DataSerializable {
```

Comment:
We should have a unit test for this new class. The main thing to test in that unit test is toData and fromData. I think the key thing to test is that the output of toData can be used as the input to fromData and the resulting changes made by fromData are equal to the original input of toData. This is not too hard. You can create a DataOutput that just writes to a byte array and then create a DataInput that reads from that byte array. You might want to add an equals method on this class or you could just ask info their maps are equal. Test toData with an empty map and a map with multiple entries.

Reply:
We added a unit test for this class